nikola/data/themes/bootstrap4/templates/gallery.tmpl

```diff
@@ -15,11 +15,36 @@
     </p>
     %endif
     %if folders:
+    % if galleries_use_thumbnail:
+    <% card_min_size = thumbnail_size + 100 %>
```

Comment:
This should probably be a setting also: `GALLERIES_CARD_HEIGHT` and `GALLERY_CARD_WIDTH`